homeassistant/components/hive/switch.py

```diff
@@ -41,16 +41,14 @@ def unique_id(self):
     @property
     def device_info(self) -> DeviceInfo | None:
         """Return device information."""
-        if self.device["hiveType"] == "activeplug":
-            return DeviceInfo(
-                identifiers={(DOMAIN, self.device["device_id"])},
-                manufacturer=self.device["deviceData"]["manufacturer"],
-                model=self.device["deviceData"]["model"],
-                name=self.device["device_name"],
-                sw_version=self.device["deviceData"]["version"],
-                via_device=(DOMAIN, self.device["parentDevice"]),
-            )
-        return None
+        return DeviceInfo(
```

Comment:
Is this change related to the library bump?

Reply:
@MartinHjelmare  no this change is to convert an existing switch that isn’t a device. To a configuration entity, as it represents the heat on demand configuration switch for the thermostat.